Makefile

```diff
@@ -78,6 +78,11 @@ test-%:
 .PHONY: test
 test: test-harness test-model_hub test-master test-agent test-webui
 
+# local frontend dev server against current DET_MASTER
+.PHONY: local
```

Comment:
nit: the target names a bit confusing to me. 🤔 don't have any good suggestions yet

Reply:
"make -C webui hosted" ... or ... "make -C webui live-front" :shrug: